lib/linux_admin/ip_address.rb

```diff
@@ -0,0 +1,25 @@
+module LinuxAdmin
+  class IpAddress
+    include Common
+
+    def ipv4
```

Comment:
Would be good to rename to address and address6 to be consistent with system utilities.